### ISSUE TITLE
change-nginx-max-body-size.config not working

### ISSUE THREAD

**Issue by fcelarino** [awsdocs/elastic-beanstalk-samples#115]
I added the contents of elastic-beanstalk-samples/configuration-files/aws-provided/instance-configuration/change-nginx-max-body-size.config into a file called 01_nginx.config inside .ebextensions in a java bundle package (zip file via ProcFile).

I'm  using Corretto 11 (running on 64bit Amazon Linux 2) and when I make a request I keep getting this error which means that apparently is not working.
<html>
<head><title>413 Request Entity Too Large</title></head>
<body>
<center><h1>413 Request Entity Too Large</h1></center>
<hr><center>nginx/1.16.1</center>
</body>
</html>

**Comment by mattr2020** [comment]
I found the current order of scripts that EB builds to launch Nginx (and other resources) places the script that this .config creates '12_add_nginx_configuration.sh' in the wrong spot.  It runs immediately after '10stop.sh' which presumably stops Nginx.  That means that when script tries to tell Nginx to reload the config (/sbin/service nginx reload), it fails because...Nginx isn't running at the point in the deployment.

Changing 12 to something great than 50 so it runs after '50start.sh' worked for me.  I just changed '12_add_nginx_configuration.sh' to '55_add_nginx_configuration.sh'.

My guess is that this script might've worked at some point in the past, but AWS has probably changed their script naming scheme when it comes to ordering.

BTW, if the instance that this ran on using the 12_... filename is still running, you'll have to manually delete that file first (ssh in and remove it from '/opt/elasticbeanstalk/hooks/appdeploy/enact/) before trying this or just re-launch the instance.

Hope this helps.
